core/src/main/java/com/linecorp/armeria/common/HttpResponse.java

```diff
@@ -320,4 +328,11 @@ static HttpResponse from(CompletionStage<? extends HttpResponse> stage) {
         subscribe(aggregator, executor);
         return future;
     }
+
+    /**
+     * Whether this is a deferred {@link HttpResponse}, created by {@link #from(CompletionStage)}.
+     */
+    default boolean isDeferred() {
```

Comment:
Just curious. When is this used? I saw its usage only in test cases, so I don't know when to use.

Reply:
Yeah it's only in the test case - maybe I can just remove those assertions, @minwoox what do you think?

Reply:
I cannot figure out when to use neither. :-)